### PR TITLE
Display some text if image is not available in the /wiki page 

### LINKED ISSUE #10776: Display some text if image is not available in the /wiki page
### 💡Format this Issue Into a First Timers Issue
This has been marked as a good candidate for becoming a [first-timers-only issue](https://code.publiclab.org/#r=all) like these, meaning that it's **simple, self-contained**, and with some extra formatting✍️, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider **reformatting it as a first-timers-only issue**, and then ping @publiclab/reviewers to get it labelled🏷. Or, if this is not your first time, **try to solve it yourself**🧑‍💻!

****
### 🤔 What you will need to know.
Display some text like `N.A` or `Not Available` in the image column if there is no image attached to the [wiki page](https://publiclab.org/wiki).
The relevent code can be found [here](https://github.com/publiclab/plots2/blob/main/app/views/wiki/_wikis.html.erb).

screenshot

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by Sakshamgupta90**
I have added a text, which shows the `None` image is not there.

Fixes #10776  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [✔️] PR is descriptively titled 📑 and links the original issue above 🔗
* [✔️] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [✔️] code is in uniquely-named feature branch and has no merge conflicts 📁
* [✔️] screenshots/GIFs are attached 📎 in case of UI updation
* [✔️] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/wiki/_wikis.html.erb [hunk 1]
```diff
@@ -25,6 +25,10 @@
             <div style="padding-bottom:30px;padding-top:10px;">
               <img src="<%= wiki.scraped_image %>" style="width:120px;border-radius:5px;">
             </div>
+            <% else %>
```
Comment (TildaDares) [inline app/views/wiki/_wikis.html.erb:28]:
```suggestion
          <% else %>
```

File: app/views/wiki/_wikis.html.erb [hunk 2]
```diff
@@ -25,6 +25,10 @@
...
+            <div>
+            <p class="font-italic">None</p>
```
Comment (TildaDares) [inline app/views/wiki/_wikis.html.erb:30]:
```suggestion
              <p class="font-italic">None</p>
```

**Comment by Sakshamgupta90** [comment]
Hi @TildaDares, Is this PR fine to merge?

File: app/views/wiki/_wikis.html.erb (hunk 1 above)
Comment (TildaDares) [reply app/views/wiki/_wikis.html.erb:28]: Same with this.

**Review by TildaDares — APPROVED** [review]
Great work @Sakshamgupta90! Thank you.

**Pull request merged**